src/commands/createWebApp/setAppWizardContextDefault.ts

```diff
@@ -35,6 +36,9 @@ export async function setAppWizardContextDefault(wizardContext: IAppServiceWizar
             // requirements.txt are used to pip install so a good way to determine it's a Python app
             wizardContext.recommendedSiteRuntime = [LinuxRuntimes.python];
 
+        } else if ((await findFilesByFileExtension(fsPath, 'csproj')).length > 0) {
+            // wizardContext.recommendedSiteRuntime = ['dotnet'];
```

Comment:
Will need to uncomment this when this PR is in https://github.com/microsoft/vscode-azuretools/pull/528

Reply:
That was merged a while ago - anything you're waiting on to fix this comment?